### ISSUE TITLE
It is wrong about the loss between the teacher and student modle

### ISSUE THREAD

**Issue by lixiaohui2020** [xiexuliunian/insightface#1]
Hi,
    I have already written the vargfacenet code, and want to talk about the distillation loss. I think the loss is about the output function of  the fc1 layer of the teacher model and the fc1 layer of the student model, not the output of the fc7 layer.

**Comment by xiexuliunian** [comment]
首先很高兴您就这个问题同我讨论，我目前也处于迷惑的状态，但根据hinton的文章里logits应该是pre softmax层的输出，不应该是fc7的输出吗？欢迎您帮我答疑解惑

**Comment by xiexuliunian** [comment]
hi ，you said that loss is about the fc1 layer of the teacher model and student model ,but I see the paper of hinton point that logits is the output of pre softmax ,isn't it the output of fc7?welcome to contact me for this question!

**Comment by lixiaohui2020** [comment]
It is not the output of fc7, and is fc1(512d)

**Comment by xiexuliunian** [comment]
you say it is fc7 ，ok ,but I want to know why?In hinton's paper said the logits should be pre softmax,why do you think fc1 is the pre softmax?
